src/components/core/Modal.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from "react";
+
+const Modal = ({ item, onClose, onSave, mode = "edit" }) => {
```

Comment:
Make the modal a dumb component, this modal cannot be reused if required showing any other content for some other purpose.

Reply:
Resolved ✅